client/lib/plans/features-list.js

```diff
@@ -757,22 +757,22 @@ export const FEATURES_LIST = {
 	},
 	[ constants.FEATURE_MALWARE_SCANNING_DAILY_AND_ON_DEMAND ]: {
 		getSlug: () => constants.FEATURE_MALWARE_SCANNING_DAILY_AND_ON_DEMAND,
-		getTitle: () => i18n.translate( 'Daily and On-demand Malware Scanning' ),
+		getTitle: () => i18n.translate( 'Daily and on-demand malware scanning' ),
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.